pyensign/iterator.py

```diff
@@ -59,8 +59,9 @@ class PublishResponseIterator(ResponseIterator):
     gRPC publish stream and executes user-defined callbacks for acks and nacks.
     """
 
-    def __init__(self, stream, on_ack=None, on_nack=None):
+    def __init__(self, stream, pending, on_ack=None, on_nack=None):
```

Comment:
Maybe we could update the docstring to explain what type `pending` is and what it is expected to contain (and how that differs from `stream`)?